documentation/webapi.md

```diff
@@ -228,6 +228,236 @@ curl -X "POST" "https://webapi.vvo-online.de/dm/trip" \
 ```
 
 
+# Query a Trip
+
+Query how to get from staion "Hauptbahnhof" (stopid 33000028) to staion
+"Bahnhof Neustadt" (stopid 33000016).
+
+## Request
+
+POST https://webapi.vvo-online.de/tr/trips
+
+### JSON body
+
+| Name               | Type        | Description       | Required |
+| ------------------ | ----------- | ----------------- | -------- |
+| `origin`           | String      | stopid of start station | yes |
+| `destination`      | String      | stopid of destination station | yes |
+| `shorttermchanges` | Bool        | unknown           | no (missing behaves like `shorttermchanges = false`) |
+| `time`             | String      | ISO8601 timestamp | no  |
+| `isarrivaltime`    | Bool        | is `time` arrival or departure | no |
```

Comment:
There are a few more params (at least the mobile client sends them as well). Sorry for the screenshot, it's the quickest way :P Unfortunately I have not played around with this a lot to find other possible values for these.
<img width="403" alt="screen shot 2017-12-09 at 17 13 44" src="https://user-images.githubusercontent.com/2625584/33797224-7b43ed6a-dd04-11e7-8941-4178c2b0e9e7.png">

Reply:
I noticed them (as you can see in the example curl request [1] below the table. Currently I don't need them, so I haven't played with them too. And I did not add them to the description table because I can only assume what they are there for. So I limited the content of the table to stuff I actually tried and what seems to be the minimum requirement to get a usable response from the API.
I thought that's enough for a starting point (You can send a request and get a useful response). If somebody needs more, they get a hint about the existence of these arguments from the curl example. And if that someone is nice, he/she will share results in the this wiki/repo.
That someone might be me some when, if I'm going to dig deeper into requesting trips. But currently I don't need more options - so no unnecessary digging.

[1]: https://github.com/kiliankoe/vvo/pull/12/commits/0f2bd9ef776612351b50e326f37e2789fe73a53b#diff-90cf528595b0f5c7456b7670ed9c7c2cR262

Reply:
Makes sense, let's worry about the details later ;)